tests/Unit/Evolution/DiscontinuousGalerkin/Test_InitializeElement.cpp

```diff
@@ -410,15 +406,34 @@ SPECTRE_TEST_CASE("Unit.Evolution.dG.InitializeElement",
       DomainCreators::Interval<Frame::Inertial>{
           {{-0.5}}, {{1.5}}, {{false}}, {{2}}, {{4}}});
 
+  test_initialize_element<Metavariables<1, false, false, tmpl::list<>>>(
+      ActionTesting::MockRuntimeSystem<
+          Metavariables<1, false, false, tmpl::list<>>>::CacheTuple{
+          std::make_unique<TimeSteppers::AdamsBashforthN>(4, false),
+          SystemAnalyticSolution{}},
+      ElementId<1>{0, {{SegmentId{2, 0}}}}, 3., 1., 1.,
+      DomainCreators::Interval<Frame::Inertial>{
+          {{-0.5}}, {{1.5}}, {{false}}, {{2}}, {{4}}});
+
   test_initialize_element<Metavariables<2, false, false, tmpl::list<>>>(
       ActionTesting::MockRuntimeSystem<
           Metavariables<2, false, false, tmpl::list<>>>::CacheTuple{
           std::make_unique<TimeSteppers::AdamsBashforthN>(4, false),
           SystemAnalyticSolution{}},
-      ElementId<2>{0, {{SegmentId{2, 1}, SegmentId{3, 2}}}}, 3., 1., 1.,
+      ElementId<2>{0, {{SegmentId{2, 0}, SegmentId{3, 2}}}}, 3., 1., 1.,
       DomainCreators::Rectangle<Frame::Inertial>{
           {{-0.5, -0.75}}, {{1.5, 2.4}}, {{false, false}}, {{2, 3}}, {{4, 5}}});
 
+  test_initialize_element<Metavariables<2, false, false, tmpl::list<>>>(
+      ActionTesting::MockRuntimeSystem<
+          Metavariables<2, false, false, tmpl::list<>>>::CacheTuple{
+          std::make_unique<TimeSteppers::AdamsBashforthN>(4, false),
+          SystemAnalyticSolution{}},
+      ElementId<2>{0, {{SegmentId{2, 0}, SegmentId{3, 8}}}}, 3., 1., 1.,
```

Comment:
{3, 8} is an illegal SegmentId ({3,7} would be the last segment)

Reply:
Whoops, yes it is.